spec/database_cleaner/configuration_spec.rb

```diff
@@ -9,131 +9,130 @@ def stub_cleaners(array)
 end
 
 RSpec.describe DatabaseCleaner::Configuration do
+  subject(:config) { described_class.new }
+
   context "orm specification" do
     it "should not accept unrecognised orms" do
-      expect { subject[nil] }.to raise_error(DatabaseCleaner::NoORMDetected)
-    end
-
-    it "should default to autodetection" do
-      require "active_record"
-      subject.strategy = :truncation
-      cleaner = subject.cleaners.values.first
-      expect(cleaner.orm).to eq :active_record
+      expect { config[nil] }.to raise_error(DatabaseCleaner::NoORMDetected)
     end
 
     it "should accept :active_record" do
-      cleaner = subject[:active_record]
+      cleaner = config[:active_record]
       expect(cleaner).to be_a(DatabaseCleaner::Base)
       expect(cleaner.orm).to eq :active_record
-      expect(subject.cleaners.values).to eq [cleaner]
+      expect(config.cleaners.values).to eq [cleaner]
     end
 
     it "should accept :data_mapper" do
-      cleaner = subject[:data_mapper]
+      cleaner = config[:data_mapper]
       expect(cleaner).to be_a(DatabaseCleaner::Base)
       expect(cleaner.orm).to eq :data_mapper
-      expect(subject.cleaners.values).to eq [cleaner]
+      expect(config.cleaners.values).to eq [cleaner]
```

Comment:
@botandrose I thought you had decided to drop support for this. So maybe it's okay to remove this here?

Reply:
Yes, good catch! Pushed. Love seeing more red in a diff!